mlir/lib/Transforms/RemoveDeadValues.cpp

```diff
@@ -115,6 +157,21 @@ static BitVector markLives(ValueRange values, RunLivenessAnalysis &la) {
   return lives;
 }
 
+// DeletionSet is used to track the Values that are scheduled for removal
+void updateDeletionSet(DenseSet<Value> &deletionSet, ValueRange range,
```

Comment:
```suggestion
static void updateDeletionSet(DenseSet<Value> &deletionSet, ValueRange range,
```

Reply:
sorry about that, thank you for noticing!